src/extensions/reactNative.js

```diff
@@ -34,7 +34,10 @@ function attach (plugin, command, context) {
     const reactNativeVersion = opts.version || parameters.options['react-native-version'] || REACT_NATIVE_VERSION
     const reactNativeTemplate = opts.template || parameters.options['react-native-template']
 
+    const perfStart = (new Date()).getTime()
```

Comment:
Would be nice to have this feature in gluegun.  https://github.com/infinitered/gluegun/issues/197